packages/contracts/contracts/libs/v0.8.x/V3FlexLib.sol

```diff
@@ -177,7 +186,13 @@ library V3FlexLib {
             // we don't want to emit data, so we emit the cid as an empty string
             _cidOrData = "";
         } else {
+            // incoming dependency type is not ONCHAIN, so we set the cid directly with either
+            // the incoming cid or string representation of the dependencyNameAndVersion
             flexProjectData.externalAssetDependencies[_index].cid = _cidOrData;
+            // clear any previously populated bytecode address
+            flexProjectData
+                .externalAssetDependencies[_index]
+                .bytecodeAddress = address(0);
```

Comment:
this is a small bug fix that clears stale data. exists in all previously deployed V3 Flex contracts, but inconsequential.

Reply:
confirmed that this does fix a bug via testing 👍 [4de1cc2](https://github.com/ArtBlocks/artblocks-contracts/pull/1518/commits/4de1cc2aa2208c9d5ef0336b8b7897c3fa45ebfb) vs. [2bba50e](https://github.com/ArtBlocks/artblocks-contracts/pull/1518/commits/2bba50e72baaa45063b88297f7e51274420d369c)